actions/publish_message_MOD.js

```diff
@@ -0,0 +1,69 @@
+module.exports = {
+  name: 'Announce/Publish Message',
```

Comment:
Rename the file to the appropriate name too (don't forget about the docs as well)
```suggestion
  name: 'Crosspost Message',
```